.github/scripts/config.sh

```diff
@@ -8,12 +8,14 @@ BINUTILS_VERSION=${BINUTILS_VERSION:-binutils-master}
 
 TARGET=${TARGET:-aarch64-w64-mingw32}
 CRT=${CRT:-msvcrt}
+TOOLCHAIN_NAME=${TOOLCHAIN_NAME:-$TARGET-$CRT}
```

Comment:
I prefer not to have crt extension for linux gnu build at least.
for mingw build I would prefer not to have extension for msvcrt build, only add it for ucrt

Reply:
I agree that for Linux targets using CRT in the package name is missleading/wrong, I'll fix that. I'd prefere to keep `msvcrt` visible for the MinGW builds for clarity and consistency reasons.